src/manifold/src/manifold.cpp

```diff
@@ -12,9 +12,12 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+#include <QuickHull.hpp>
 #include <algorithm>
+#include <cstdio>
```

Comment:
necessary?

Reply:
Nope, just missed cleaning it up after doing some debugging.